notebooks/2021-08-16/Sun Angles.ipynb

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
<p>Nice! I hadn't seen PySolar before - it looks pretty perfect!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openclimatefix/nowcasting_dataset/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>